src/Microsoft.DocAsCode.Common/FileAbstractLayer/ManifestFileHelper.cs

```diff
@@ -59,5 +59,41 @@ public static bool AddFile(this Manifest manifest, string sourceFilePath, string
             }
             return false;
         }
+
+        public static void RemoveLinks(this Manifest manifest, string manifestFolder)
```

Comment:
Unlink?

---
In reply to: [95950368](https://github.com/dotnet/docfx/pull/1172#discussion_r95950368) [](ancestors = 95950368,95947362)